src/OFL-1.1.xml

```diff
@@ -6,6 +6,11 @@
          <crossRef>http://www.opensource.org/licenses/OFL-1.1</crossRef>
       </crossRefs>
       <notes>This license was released: 26 February 2007.</notes>
+      <optional>
```

Comment:
`copyrightText` is already optional ([guideline 10.1.1][1]).  I'm in favor of explicit `<optional>` tags for optional copyright text, but that's not where this repo is now.  I think we should shift this line after `<copyrightText>` for this PR and move it back to here it if/when the whole repository moves to use explicit `<optional>` tags for `<copyrightText>`.

[1]: https://spdx.org/spdx-license-list/matching-guidelines

Reply:
Thanks, agreed on all points.